geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,25 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+<div class="merge-document-label">
+    <strong><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}">{{ document }}</a></strong>
+    <p>PGPID {{ document.id }}</p>
+    {% if document.description %}
+        <p>
+            <dt>Description</dt>
+            <dd>{{ document.description }}</dd>
+        </p>
+    {% endif %}
```

Comment:
I know Rachel didn't request it, but the merge function includes languages, tags, notes, and needs review — and I suspect some of them at least would be helpful here; in many cases there are documents now that have notes indicating which records they should be merged with.  Should we check with her about which of those should be added to the display?